app/flux/components/CategoryListItem.react.js

```diff
@@ -0,0 +1,43 @@
+var { Link } = require('react-router');
```

Comment:
Module syntax was benched for ES6, this is just simply getting the prototype class Link from the module.